### PR TITLE
Improve Bullet

### PR DESCRIPTION
**Pull request by GabrielRavier**
This improves bullet-related files with code style improvements

### CODE REVIEW

File: src/BulHit.cpp [hunk 1]
```diff
@@ -339,12 +339,12 @@ void HitBulletMap()
...
 		atrb[3] = GetAttribute(x + 1, y + 1);
 
-		//Clear hit tiles
+		// Clear hit tiles
 		gBul[i].flag = 0;
 
 		if (gBul[i].bbits & 4)
 		{
-			// For some reason a 'continue' just doesn't produce the same assembly
+			// A "continue" doesn't produce the same assembly, likely because that would jump to the generated "continue" label instead of the "endif" label
```
Comment (Clownacy) [inline src/BulHit.cpp]: I don't get the point of this comment. I already know what the assembly looks like.

File: src/Bullet.cpp [hunk 2]
```diff
@@ -222,14 +222,16 @@ void ActBullet_Frontia1(BULLET *bul)
 		if (bul->ani_no > 3)
 			bul->ani_no = 0;
 
-		RECT rcLeft[4] = {
+		RECT rcLeft[4] =
+		{
```
Comment (Clownacy) [inline src/Bullet.cpp]: Same problem as the other pull request.

File: src/BulHit.cpp (hunk 1 above)
Comment (GabrielRavier) [reply src/BulHit.cpp]: The reason for that change was that people familiar with how MSVC generates aren't the only people who are going to be looking at this

Comment (Clownacy) [reply src/BulHit.cpp]: I don't see why people trying to make things ASM-accurate wouldn't be able to just check the code themselves.

Comment (Clownacy) [reply src/BulHit.cpp]: I already said why it generates different code. You're saying how.

Comment (GabrielRavier) [reply src/BulHit.cpp]: I mean my problem with the old comment is mostly that it has "for some reason" which makes it look like the reason for this is unknown. Really I don't really see how saying in a more detailed manner how MSVC generates different code in this situation is bad

**Comment by GabrielRavier** [comment]
I think I corrected it

File: src/BulHit.cpp (hunk 1 above)
Comment (Clownacy) [reply src/BulHit.cpp]: The 'for some reason' was aimed at the continue not working, meaning Pixel did something different that I can't figure out, besides just having empty brackets.

Comment (GabrielRavier) [reply src/BulHit.cpp]: Oh, ok. Getting back with a modified comment

File: src/Bullet.cpp [hunk 3]
```diff
@@ -1300,7 +1300,8 @@ void ActBullet_Spine(BULLET *bul)
 			{232, 0, 240, 8},
 		};
 
-		RECT rcDown[2] = {
+		RECT rcDown[2] =
+		{
```
Comment (Clownacy) [inline src/Bullet.cpp]: Missed one

**Comment by GabrielRavier** [comment]
I corrected the RECT thing I forgot. Also I modified the continue stuff comment so I closed the old review (it doesn't update).

**Pull request merged**